admin_guide/manage_scc.adoc

```diff
@@ -57,10 +57,9 @@ restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs
 [[examining-a-security-context-constraints-object]]
 == Examining a Security Context Constraints Object
 
-To examine a particular SCC, use `oc get`, `oc describe`, `oc export`, or `oc
-edit`. For example, to examine the *restricted* SCC:
+To examine a particular SCC, use `oc get`, `oc describe`, `oc export`, `oc
+explain`, or `oc edit`. For example, to examine the *restricted* SCC:
```

Comment:
`oc explain` isn't related to "particular" SCC. Instead it shows possible properties that all SCC have.